compiler/src/dotty/tools/dotc/cc/CheckCaptures.scala

```diff
@@ -977,7 +971,7 @@ class CheckCaptures extends Recheck, SymTransformer:
               .getOrElse(cls, cls.info.decls.toList)           // pick all symbols in class scope for other classes
               .flatMap(classifiersOfFreshInType)
           if cls.typeRef.isMutableType then
-            fieldClassifiers = defn.Caps_Mutable :: fieldClassifiers
+            fieldClassifiers = defn.AnyClass :: fieldClassifiers
```

Comment:
Is there a reasion to add AnyClass here as opposed to just not do anything?

Reply:
Yes, without it we would not get a `cap` in the capture set of the constructor at all.  But in fact there's a possible refinement, I'll push that now.